src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java

```diff
@@ -1,236 +1,243 @@
-    int validHrefWithWhiteSpaces = 54;
+  }
+
+  /** test method pattern */
+  void ALL_UPPERCASE_METHOD() {}
+
+  /** test illegal constant * */
+  private static final int BAD__NAME = 3;
+
+  // A very, very long line that is OK because it matches the regexp "^.*is OK.*regexp.*$"
+  // long line that has a tab ->    <- and would be OK if tab counted as 1 char
+  // tabs that count as one char because of their position ->    <-   ->    <-, OK
+
+  /** some lines to test the column after tabs */
+  void violateColumnAfterTabs() {
+    // with tab-width 8 all statements below start at the same column,
+    // with different combinations of ' ' and '\t' before the statement
+    int tab0 = 1;
+    int tab1 = 1;
+    int tab2 = 1;
+    int tab3 = 1;
+    int tab4 = 1;
+    int tab5 = 1;
+  }
+
+  // MEMME:
+  /* MEMME: a
+   * MEMME:
+   * OOOO
+   */
+  /* NOTHING */
+  /* YES */
+  /* MEMME: x */
+  /* YES!! */
+
+  /** test long comments * */
+  void veryLong() {
+    /*
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    blah blah blah blah
+    enough talk */
+  }
+
+  /**
+   * @see to lazy to document all args. Testing excessive # args
+   */
+  void toManyArgs(
+          int aArg1,
+          int aArg2,
+          int aArg3,
+          int aArg4,
+          int aArg5,
+          int aArg6,
+          int aArg7,
+          int aArg8,
+          int aArg9) {}
+
+  /**
+   * Very long url with https:
+   * https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java
+   */
+  String https = "200 OK";
+
+  /**
+   * Very long url with http:
+   * http://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java
+   */
+  String http = "200 OK";
+
+  // Very long url with ftp:
+  // ftp://ftp.example.com/areallyyyyyyyyyyyylongggggggggggggggggggggggurlllll.text
+  int ftp = 0;
+
+  // Very long url with invalid href:
+  // href="www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search
+  // violation above 'Line is longer than 100 characters (found 111).'
+  int invalidHref = 88;
+
+  // Very long url with valid href:
+  // href="www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search"
+  int validHref = 54;
+
+  // Very long url with valid href: href    = "www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search"
+  int validHrefWithWhiteSpaces = 54;
```

Comment:
FYI: Formatter formatted this line to something like:

```java
  // Very long url with valid href: href    =
  // "www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search"
```

but this line was essential to test the section properly so I reverted formatter's changes